operator/plugin_test.go

```diff
@@ -261,18 +261,353 @@ parameters:
   path:
     label: Path
     description: The path to a thing
-    type: []
+    type: {}
 pipeline:
 `,
 		},
 		{
 			name:      "empty_parameter",
+			expectErr: true,
+			template: `version: 0.0.0
+title: My Super Plugin
+description: This is the best plugin ever
```

Comment:
Can we change the test titles and descriptions to something more professional/generic like "Test Plugin" and "This is a test plugin."